builder/update_test.go

```diff
@@ -4,595 +4,787 @@ import (
 	"database/sql"
 	"fmt"
 	"testing"
-	"time"
 
 	"github.com/lib/pq"
-	"github.com/stretchr/testify/require"
 
 	"github.com/ulule/loukoum"
 	"github.com/ulule/loukoum/builder"
 	"github.com/ulule/loukoum/format"
 )
 
-func TestUpdate_Set_Undefined(t *testing.T) {
-	is := require.New(t)
-
-	// Ensure that update panics if SET clause is empty.
-	Failure(is, func() builder.Builder {
-		return loukoum.Update("table")
-	})
-	Failure(is, func() builder.Builder {
-		return loukoum.Update("table").Set("")
-	})
-}
-
-func TestUpdate_Set_Map(t *testing.T) {
-	is := require.New(t)
-
-	// Variadic with built-in Map type
+var updatetests = []BuilderTest{
```

Comment:
Could you add https://github.com/ulule/loukoum/commit/3844b1858d08af831400dcf7f5ea0cd5536dd153#diff-264d19b3ccabb03bc549d7c023af3c18R471 ? Thx